INN/umbrella-caribbean
# COVID series landing page updates

## Changes

This pull request makes the following changes:

- Updates the COVID landing page changes made in #116 
   - Adds "Read more" button to all posts on homepage, author pages, widgets, etc.
   - Styles that new read more button
   - Updates grid for new series child landing page
   - A few other misc. style updates for the new series child landing page

## Why



For the COVID series landing page

## Testing/Questions

Features that this PR affects:

- COVID series landing page
- Posts being displayed by `widget-content.php`, `archive-category-primary-feature.php`, and `home-top.php` partials


Questions that need to be answered before merging:


Steps to test this PR:

1. View all types of pages and widgets to make sure the new read more button works as expected
2. View the COVID landing page and make sure its styling looks ok

## Additional information

INN Member/Labs Client requesting: (if applicable)
